apps/nuxt3-ssr/components/harmonization/HarmonizationListPerVariable.vue

```diff
@@ -1,16 +1,51 @@
 <script setup lang="ts">
 import type { HarmonizationStatus } from "~/interfaces/types";
 
-defineProps<{
-  cohortsWithMapping: { cohort: { id: string }; status: HarmonizationStatus }[];
+const props = defineProps<{
+  cohortsWithMapping: {
+    cohort: { id: string };
+    status: HarmonizationStatus | HarmonizationStatus[];
+  }[];
 }>();
+
+const aggregatedHarmonizationStatus = computed(() => {
+  return props.cohortsWithMapping
+    .map((cwm) => {
+      const status = Array.isArray(cwm.status) ? cwm.status : [cwm.status];
+      return { cohort: cwm.cohort, status };
+    })
+    .map((statusPerCohort) => {
+      if (
+        statusPerCohort.status.includes("partial") ||
```

Comment:
this logic, this has been checked with Marije?

Reply:
yes , she suggested to replace with grid , i created a issue for that